### ISSUE TITLE
Swimming activities are not syncd

### ISSUE THREAD

**Issue by petergebala** [cpfair/tapiriik#237]
I am connected to GarminConnect, Runkeeper and Endomondo.
I track swimming activity on Garmin Watch and then I am syncing activites by tapiriik.
Synchronization with Runkeeper is working fine, but Endomondo don't see swimming activites. 
Other sports tracks are working fine.

**Comment by iProkurator** [comment]
The same problem. I see checkbox under Endomondo (https://tapiriik.com/activities), but I dont see it in the Endo..

**Comment by randybb** [comment]
Yup, from GC to Strava it works and the integrated synchronization in Endomondo can synchronize it with GC, so it should be possible with their API as well.
